### PR TITLE
fix history

### PR DESCRIPTION
**Pull request by rbs333**

### CODE REVIEW

File: examples/dbpedia/process_data.ipynb
Comment (rbs333) [inline examples/dbpedia/process_data.ipynb:1]: This needs to get cleaned up / removed

File: examples/dbpedia/optimization.ipynb
Comment (rbs333) [inline examples/dbpedia/optimization.ipynb:1]: I'm not sure if we want to keep this as an example or not

File: examples/dbpedia/dbpedia_study_config.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+input_data_type: "json" # currently only json but considering alternatives
+raw_data_path: "data/dbpedia_250k.json" # path to data files for easy read
```
Comment (rbs333) [inline examples/dbpedia/dbpedia_study_config.yaml:2]: These dataset should be uploaded to S3 if we want them as part of the public examples

File: label_app/main.py [hunk 2]
```diff
@@ -11,29 +11,37 @@
...
 
 # Change to schema config that represents your data as needed
 # make sure model used matches the dimensions of the schema
 # if in .env file will load from there otherwise will default to the provided
-SCHEMA_PATH = os.getenv("SCHEMA_PATH", "label_app/schema/cars_schema.yaml")
+SCHEMA_PATH = os.getenv("SCHEMA_PATH", "label_app/schema/index_schema.yaml")
+EMBEDDING_PROVIDER = os.getenv("EMBEDDING_PROVIDER", "hf")
 EMBEDDING_MODEL = os.getenv("EMBEDDING_MODEL", "sentence-transformers/all-MiniLM-L6-v2")
-REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:6379/0")
+REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:6381/0")
```
Comment (rbs333) [inline label_app/main.py:29]: This should change back to the default

File: optimize/study.py [hunk 3]
```diff
@@ -60,6 +60,12 @@ def update_metric_row(eval_obj):
     METRICS["retriever"].append(str(eval_obj.retriever.__name__))
 
 
+def persist_metrics(client, eval_obj, study_id):
```
Comment (rbs333) [inline optimize/study.py:63]: persist metrics as the study goes in case of failure don't loose all runs

**Review by justin-cechmanek — APPROVED** [review]
I would keep the examples. Generally, I'd say keep large datasets in our redis-ai-resources S3 bucket, but for docs like our mazda_3.pdf and the associated json chunks I think those are small enough to keep in git.

**Pull request merged**